.github/scripts/cmd/cmd.py

```diff
@@ -262,9 +262,10 @@ def main():
         command = f"cargo +nightly fmt"
         print(f'Formatting with `{command}`')
         nightly_status = os.system(f'{command}')
+        zepter_status = os.system('zepter run default')
```

Comment:
Why default? The CI uses check as `zepter run check` and it should find and use `.config/zepter.yaml` config file,
so we should do the same here.

```suggestion
        zepter_status = os.system('zepter run')
```

Reply:
@ggwpez what is the recommended order? zepter then taplo or taplo then zepter?

Reply:
First zepter then taplo. You can just do `zepter`, no need to add any arguments.